index.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Home page</title>
+
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+      integrity="sha512-XXXXX"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.7.0/css/bootstrap.min.css"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    />
+  </head>
+  <body>
+    <!-- Header -->
+    <header class="header">
+      <ul class="header-list">
+        <li class="header-items">
+          <i class="fa-brands fa-facebook"></i>
+        </li>
+        <li class="header-items">
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li class="header-items">English</li>
+        <li class="header-items">My Page</li>
+        <li class="header-items">Logout</li>
+      </ul>
```

Comment:
According to the design, the top bar doesn't have a bottom left and right rounded edge, kindly remove it and make it a regular square edge to make your design match the template. Also, consider respecting the correct background color which is #272a31

<img width="949" alt="round" src="https://github.com/Zafron047/Capstone_Project/assets/84930355/d59b91a6-f868-4f49-92b2-7f98517078f7">

Reply:
Border Radius remove.

Complete.